fsthttp/backend.go

```diff
@@ -0,0 +1,84 @@
+package fsthttp
+
+import (
+	"time"
+
+	"github.com/fastly/compute-sdk-go/internal/abi/fastly"
+)
+
+type TLSVersion uint32
+
+const (
+	TLSVersion1   TLSVersion = 0
```

Comment:
```suggestion
	TLSVersion1_0 TLSVersion = 0
```

I recommend being explicit about calling it 1.0, since that seems to be the convention and there are incompatibilities between each minor version.